hs_core/templates/pages/genericresource.html

```diff
@@ -975,6 +989,78 @@ <h4 class="modal-title">Confirm file deletion</h4>
                                 </div>
                                 {% endif %}
 
+                                  <!-- ========= CONNECTIONS TAB ========= -->
+                                {% if connections %}
+                                    <div class="tab-pane" id="connectionsTab">
+                                         {% if connections.collected_by %}
+                                                <h4><strong>Belongs to Collections:</strong></h4>
+                                                <hr style="margin-top: 0;margin-bottom: 2px">
+                                                <table id="collected-by-table" class="table-hover table-striped resource-custom-table" width="100%">
+                                                    <thead>
+                                                        <tr>
+                                                            <th>Title</th>
+                                                            <th>Owners</th>
+                                                            <th>Sharing Status</th>
+                                                            <th>My Permission</th>
+                                                        </tr>
+                                                    </thead>
+                                                    <tbody>
+                                                        {% for res in connections.collected_by %}
```

Comment:
I am happy for you to implement what you think is best.  Part of what I was thinking of was what is editable by a user and what is not, and that it may be confusing to a user if some relationships can be edited and others just appear.  Whether it is harder or easier to add to an existing tab or make a new tab is, for me, a lesser consideration than presenting this information in a clear and unambiguous way for the user.  I'd prefer the focus to be on the user (even if a little harder to implement - which it may not be in this case).  I agree that adding more tabs does make it more complicated for a user, so if the information can be clearly presented on the references tab, that would be good.